src/deq_eid/main.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+"""
+Run the deq_eid script as a cloud function.
+"""
+import json
+import logging
+import sys
+from datetime import datetime
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from types import SimpleNamespace
+
+import arcgis
+from arcgis.features import GeoAccessor
+from arcgis.gis._impl._content_manager import SharingLevel
+from palletjack import extract, load
+from supervisor.message_handlers import SendGridHandler
+from supervisor.models import MessageDetails, Supervisor
+
+#: This makes it work when calling with just `python <file>`/installing via pip and in the gcf framework, where
+#: the relative imports fail because of how it's calling the function.
+try:
+    from . import config, helpers, version
+except ImportError:
+    import config
+    import helpers
+    import version
```

Comment:
**question**: is this still applicable with the docker build in cloud run?

Reply:
No! I tested and it worked great without this workaround. Nice catch.